corehq/apps/translations/models.py

```diff
@@ -129,8 +129,8 @@ class TransifexBlacklist(models.Model):
     """
 
     domain = models.CharField(max_length=255)
-    app_id = models.CharField(max_length=32)
-    module_id = models.CharField(max_length=32)
+    app_id = models.CharField(max_length=255)
+    module_id = models.CharField(max_length=255)
```

Comment:
I found out while testing that these aren't actually UUIDs, and are actually 40 characters for some reason. I'll be following up on a separate PR to change that